ts/pages/staking/home.tsx

```diff
@@ -17,6 +17,8 @@ export const StakingIndex: React.FC<StakingIndexProps> = props => {
                 titleMobile="Start staking your tokens"
                 description="Use one pool of capital across multiple relayers to trade against a large group"
                 figure={<CFLMetrics />}
+                // TODO(kimpers): Add correct video for staking portal
+                videoId="c04eIt3FQ5I"
```

Comment:
Need to update this with the real video once I have that. Used the homepage video as a placeholder